### PR TITLE
feat: Add new privacy type

### PR DESCRIPTION
**Pull request by ChaituVR**
Towards https://github.com/snapshot-labs/workflow/issues/277

### Summary
- Add new type of privacy to space settings `"any"`
- Allow privacy in proposal payload (either `""` or `"shutter"`) - on sequencer we handle it like here https://github.com/snapshot-labs/snapshot-sequencer/pull/467
- Allow privacy in update-proposal

### How to test
- Run `yarn test`
- Try changing values in example.json files

### CODE REVIEW

File: src/schemas/space.json [hunk 1]
```diff
@@ -335,7 +332,7 @@
             },
             "privacy": {
               "type": "string",
-              "enum": ["", "shutter"]
+              "enum": ["", "shutter", "author-selection"]
```
Comment (ChaituVR) [inline src/schemas/space.json:335]:
```suggestion
              "enum": ["", "shutter", "any"]
```

File: test/examples/space.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
   "voting": {
     "delay": 2592000,
     "period": 15552000,
-    "quorum": 100
+    "quorum": 100,
+    "privacy": "author-selection"
```
Comment (ChaituVR) [inline test/examples/space.json:37]:
```suggestion
    "privacy": "any"
```

File: src/schemas/space.json [hunk 3]
```diff
@@ -324,9 +324,6 @@
               "type": "string",
               "enum": ["rejection"]
             },
-            "blind": {
```
Comment (wa0x6e) [inline src/schemas/space.json:327]: How is this related ?

Comment (ChaituVR) [reply src/schemas/space.json:327]: Looks like we are not using this field anymore. i assume we were using this field long ago to check if it shutter or not (need to remove on hub as well)

Comment (ChaituVR) [reply src/schemas/space.json:327]: maybe should i create a seperate PR?

Comment (wa0x6e) [reply src/schemas/space.json:327]: > maybe should i create a seperate PR?

Better, yes

**Review by bonustrack — APPROVED** [review]
utACK

**Review by wa0x6e — APPROVED** [review]
utAck
